core/envelope-sim/src/main/kotlin/fr/sncf/osrd/envelope_sim/EnvelopeSimContext.kt

```diff
@@ -28,19 +29,42 @@ constructor(
          * *after* the end of the path (or right at the end).
          */
         val dangerPointOffsets: List<Offset<TravelledPath>>,
+        /**
+         * List of ETCS-block detector offsets for every block on the path. Starts at the start of
+         * the path, can end after the end of the path.
```

Comment:
Currently we don't have any data with ETCS detectors detached from signal, but we wouldn't consider them, and it's the desired behavior, so I would explicit it a bit more:
```suggestion
         * List of block-delimiting detector (block entry/exit) offsets for every ETCS-block on the path. Starts at the start of
         * the path, can end after the end of the path.
```